nix/internal/unix.nix

```diff
@@ -163,14 +166,15 @@ in
     in
       (import inputs.flake-compat {
         src =
+          if targetSystem != "aarch64-darwin" && targetSystem != "aarch64-linux"
```

Comment:
This looks like a merge/rebase artifact – wrong conflict resolution? The line is repeated.